subworkflows/local/mapseq_asv_krona_swf.nf

```diff
@@ -70,7 +70,8 @@ workflow MAPSEQ_ASV_KRONA {
                                       }
         
         MAKE_ASV_COUNT_TABLES(
-            final_asv_count_table_input
+            final_asv_count_table_input,
+            krona_tuple[4]
```

Comment:
shouldn't this be korna_tuple -> it[4]?

Reply:
this isn't needed, krona_tuple is a 'constant' tuple defined for example here:

 https://github.com/EBI-Metagenomics/amplicon-pipeline/blob/1ce86a29a8c94b66300ae4e429e4d3e35adf4e04/workflows/pipeline.nf#L109